ci-operator/step-registry/ignition/bastionhost/ignition-bastionhost-commands.sh

```diff
@@ -85,6 +85,12 @@ EOF
 cat > ${workdir}/squid.conf << EOF
 auth_param basic program /usr/lib64/squid/basic_ncsa_auth /etc/squid/passwords
 auth_param basic realm proxy
+#prfsr90
+delay_pools 1
+delay_class 1 1
+#delay_parameters 1 10000000/10000000 -1/-1 -1/-1
+delay_parameters 1  8750000/8750000 8750000/8750000 8750000/8750000
+http_port 3128
```

Comment:
If the change is for your debugging purpose, the change looks good to me.

Reply:
we would also want to install the rosa-classic cluster on this @jianlinliu 

acl authenticated proxy_auth REQUIRED
acl CONNECT method CONNECT
http_access allow authenticated
http_port 3128
dns_v4_first on

hope these code lines are fine ?

the debug was to check if we are limiting the bandwidth that we have set on the proxy and proxy has started good.

Reply:
Please let know if I am good to rehearse now. Thank you!

Reply:
okay to rehearse.

Reply:
thank you